assets/openshift-apiserver/config.yaml

```diff
@@ -15,6 +15,7 @@ aggregatorConfig:
 apiServerArguments:
   minimal-shutdown-duration:
   - 3s
+  audit-log-maxbackup: "10"
```

Comment:
Per https://github.com/openshift/apiserver-library-go/blob/5cdb70a1e65b6bcabb3b897e38287ed2c8ed77d1/pkg/configflags/audit.go#L34, `audit-log-maxbackup` is equivalent to `maximumRetainedFiles`.  So this PR is not necessary.  I think we need this fixed in Hypershift, not the IBM ROKS Toolkit.

Reply:
Sounds good, closing the PR based on the above comment.